src/test/java/fi/otavanopisto/kuntaapi/test/server/integration/ptv/PrintableFormServiceChannelInTestsIT.java

```diff
@@ -0,0 +1,185 @@
+    V7VmOpenApiPrintableFormChannelInBase ptvInResource = getPtvServiceChannelMocker().readEntity(AbstractPtvMocker.PTV_IN_API, ptvId, V7VmOpenApiPrintableFormChannelInBase.class);
+    V7VmOpenApiPrintableFormChannel ptvOutResource =  getPtvServiceChannelMocker().readEntity(AbstractPtvMocker.PTV_OUT_API, ptvId, V7VmOpenApiPrintableFormChannel.class);
+    
+    getPtvServiceChannelMocker().mockPrintableFormPut(ptvId, ptvOutResource);
+    
+    givenReadWrite()
+      .body(kuntaApiResource)
+      .contentType(ContentType.JSON)
+      .put(SERVICE_CHANNEL_FIND_PATH,kuntaApiChannelId)
+      .then()
+      .assertThat()
+      .statusCode(200);
+
+    getPtvServiceChannelMocker().verifyPrintableForm(ptvId, ptvInResource);
+  }
+  
+  @Test
+  public void updatePrintableFormServiceChannelChanges() throws IOException, InterruptedException {
+    String ptvId = TestPtvConsts.PRINTABLE_FORM_SERVICE_CHANNELS[0];
+    String organizationId = getOrganizationId(0);
+    
+    String kuntaApiChannelId = getPrintableFormChannelId(0, TestPtvConsts.PRINTABLE_FORM_SERVICE_CHANNELS.length);
+    
+    grantOrganizationPermission(AccessType.READ_WRITE, organizationId, ClientOrganizationPermission.UPDATE_SERVICE_CHANNELS);
+
+    PrintableFormServiceChannel kuntaApiResource = getPrintableFormServiceChannel(0, TestPtvConsts.PRINTABLE_FORM_SERVICE_CHANNELS.length);
+    
+    kuntaApiResource.setAreas(Arrays.asList(createArea("Municipality", "12345", createLocalizedValue("en", "Examplia"))));
+    kuntaApiResource.setAreaType("AreaType");
+    kuntaApiResource.setAttachments(createAttachments("en", "PDF", "https://www.example.com", "Example PDF", "PDF file for testing"));
+    kuntaApiResource.setChannelUrls(createLocalizedValue("en", "URL", "https://www.example.com/channelurl"));
+    kuntaApiResource.setDeliveryAddress(createAddressFreeText(createLocalizedValue("en", "Far far away")));
+    kuntaApiResource.setDescriptions(createLocalizedValue("en", "Description", "Changed Description"));
+    kuntaApiResource.setFormIdentifier(createLocalizedValue("en", "Example form 1234"));
+    kuntaApiResource.setFormReceiver(createLocalizedValue("en", "Example receiver"));
+    kuntaApiResource.setNames(createLocalizedValue("en", "Name", "Changed Name"));
+    kuntaApiResource.setSupportEmails(createEmails("en", "fake@example.com"));
+    kuntaApiResource.setSupportPhones(Arrays.asList(
+      createPhone("en", "Phone", "+358", "12345-FAKE", "Charged", "Testing", false, "Test phone"),
+      createPhone("en", "Fax", "+258", "54321-FAKE", "Free", "Testing fax", true, "Test fax")
+    ));
+    
+    V7VmOpenApiPrintableFormChannelInBase ptvInResource = getPtvServiceChannelMocker().readEntity(AbstractPtvMocker.PTV_IN_API, ptvId, V7VmOpenApiPrintableFormChannelInBase.class);
+    ptvInResource.setAreas(Arrays.asList(createArea("Municipality", "12345")));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)